# Individual tire pressure information as reported by vehicle

std_msgs/Header header

uint8 front_left_tire_pressure     # Measured front left tire pressure in kPa in the range [0,1020]
uint8 front_right_tire_pressure    # Measured front right tire pressure in kPa in the range [0,1020]
uint8 rear_left_tire_pressure      # Measured rear left tire pressure in kPa in the range [0,1020]
uint8 rear_right_tire_pressure     # Measured rear right tire pressure in kPa in the range [0,1020]
